EIPS/eip-867.md

```diff
@@ -0,0 +1,190 @@
+
+If accepted, an ERP could easily compile the block at while the changes are to take place, and the source of the actions (which would be the output of the script, in a standardized object output). These can be bundled with the client for seamless execution.
+
+
+## ERP Approval Process
+Approval of any ERP will require a mandatory minimum of 30-days between the time the ERP is submitted (or substantially modified) and adoption by clients, and ERPs affecting more funds or more addresses may prompt a considerably longer review period.  In practice, ERPs will often require more time because the will likely be bundled into other client upgrades that are on their own schedule.  However, the 30-day mandatory review period  guarantees a period for comment and auditing and prevents any last minute inclusions from being rushed into an upcoming upgrade.  
+
+Once an ERP is published (via a pull request made to the EIP repository), the approval process begins.  The author is encouraged to seek feedback from the following groups:
+
+- People or organizations who are directly affected by the issue (e.g. entities who will be refunded or were associated with the original incident)
+- People or organizations that have a demonstrable history within the Ethereum community that are unaffected by the issue but may serve as relatively impartial adjudicators
+
+ERPs that have strong support from both groups of people will be much more likely to succeed.  The first group, although not impartial, is critical to determine that among the directly affected parties there is broad agreement about the desired outcome.  The second group is critical to help verify (a) the conditions defined in the verification script accurately represent the human readable version, (b) that the verification script generates the State Change Object submitted with the ERP, and (c) that the people/orgs representing the “affected parties” is representative.  
+
+Positive, explicit, and broad support from both groups will be critical to the approval process; a lack of negative feedback is not sufficient.  It is the sole responsibility of the ERP author and affected parties to find support. Feedback that is not substantive should be dismissed, and feedback for the ERP process as a whole should be directed to this meta-EIP.  Community members, in general, are encouraged to reference these guidelines to ensure the process remains focused, objective, and expedient.  
```

Comment:
Who chooses the feedback that is not substantive, and is to be dismissed?

Reply:
Each person individually.  We were attempting to define a community "norm".  

Feedback such as "screw these guys, I hate them" is not substantive.  

The following would be substantive "The ETH that is claimed to be frozen in this contract can be accessed by calling XYZ method from this account ABC.  The author has demonstrated that they still have access to account ABC by providing a signed message.  Therefore, the funds are not actually frozen".

Reply:
@pirapira suggested "Somebody who knows ethereum" as a standard.

Maybe "People **who know ethereum** or organizations that are unaffected by the issue but may serve as relatively impartial reviewers and have the technical ability to confirm or reject the factual claims made in the ERP"  would be a decent compromise?